__tests__/entity.update.unit.test.js

```diff
@@ -17,7 +17,7 @@ const TestEntity = new Entity({
     test_string: { type: 'string', coerce: false, default: 'test string' },
     test_string_coerce: { type: 'string' },
     test_number: { type: 'number', alias: 'count', coerce: false },
-    test_number_coerce: { type: 'number', default: 0 },
```

Comment:
This was causing changes to the generated expressions once coercion was happening for the defaulted value, so I removed it. The alternative it seemed was to fixup the other strings to expect that coerced type